lib/appsignal/phoenix.ex

```diff
@@ -57,6 +57,13 @@ defmodule Appsignal.Phoenix do
     # Submit error
     {r.__struct__, Exception.message(r), stack, Map.get(r, :conn, conn)}
   end
+  def extract_error_metadata(r, conn, stack) when is_binary(r) do
+    extract_error_metadata(RuntimeError.exception(r), conn, stack)
+  end
+  def extract_error_metadata(r, conn, stack) when is_atom(r) do
+    extract_error_metadata(r.exception([]), conn, stack)
```

Comment:
I don't get `r.exception([])` here.. `r` is an atom at this point; calling a function on it will most likely fail, or not? Depending on whether `r` is a module..